src/Writers/HTMLWriter.jl

```diff
@@ -464,6 +500,20 @@ module RD
             """
         ))
     end
+    function mathengine!(r::RequireJS, engine::MathJax3)
+        push!(r, Snippet([], [],
+            """
+            window.MathJax = $(json_jsescape(engine.config, 2));
+
+            (function () {
+                var script = document.createElement('script');
+                script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-svg.js';
```

Comment:
I would prefer if we'd hard-code the exact version here, to avoid any potential breakage.

I would also prefer if we'd CDN this [from cdnjs](https://cdnjs.com/libraries/mathjax). We use that for the other assets (except for fonts) and it's good to minimize the number of potential points of failure.